### ISSUE TITLE
audit-avt1: code snippet comments - text contrast violation

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon-website#2066]
## Description
Currently our code snippet comments do not meet Accessibility text contrast requirements

- Text contrast of 1.16 with its background is less than the WCAG AA minimum requirements for text of size 16px and weight of 400

- The contrast ratio of text with its background must meet WCAG 2.1 AA requirements

<img width="1400" alt="Screen Shot 2020-12-17 at 11 47 32 AM" src="https://user-images.githubusercontent.com/32720851/103800037-7f93f080-5011-11eb-892a-5b4972090798.png">
